sdk/eventhub/event-hubs/test/perf/track-1/receive.spec.ts

```diff
@@ -0,0 +1,150 @@
+/*
+# Overview
+Measures the maximum throughput of `receiver.receive()` in package `@azure/service-bus`.
+
+# Instructions
+1. Create a Service Bus namespace with `Tier=Premium` and `Messaging Units=4`.  It is recommended to use the largest possible namespace to allow maximum client throughput.
+2. Create a queue inside the namespace.
+3. Set env vars `SERVICE_BUS_CONNECTION_STRING` and `SERVICE_BUS_QUEUE_NAME`.
+4. This test presumes that there are messages in the queue.
+5. `ts-node receive.ts [maxConcurrentCalls] [totalMessages] [isReceiveAndDelete]`
+6. Example: `ts-node receive.ts 1000 1000000 false`
+ */
+
+import { EventHubClient, EventPosition, EventData } from "@azure/event-hubs";
+import { getEnvVar } from "@azure/test-utils-perfstress";
+import moment from "moment";
+import { delay } from "@azure/core-amqp";
+
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const _start = moment();
+
+let _messages = 0;
+
+const connectionString = getEnvVar("EVENTHUB_CONNECTION_STRING");
+const eventHubName = getEnvVar("EVENTHUB_NAME");
+const consumerGroup = getEnvVar("CONSUMER_GROUP_NAME");
+
+async function main(): Promise<void> {
+  const maxBatchSize = process.argv.length > 2 ? parseInt(process.argv[2]) : 10;
```

Comment:
I don't think this is needed since you're calling the streaming receiver, not the receiveBatch method.